content/A take on empowering ActiveRecord.md

```diff
@@ -0,0 +1,63 @@
+# A take on empowering ActiveRecord
+
+I think we are quite underpowered when it comes to doing queries on Rails, having this in mind I proposed my opinion on how we could improve this and made a PR. I also wanted to share it here to know what others think and maybe get some traction.
+
+## Querying with Ruby blocks
```

Comment:
maybe before this section you could describe the problem that the PR solves. e.g. how many times have you needed to use strings when querying with ActiveRecord like `Post.where('created_at > ?', 2.days.ago)` wouldn't it be great to be able to do it in plain ruby?